ecs-logs-collector.sh

```diff
@@ -496,6 +499,43 @@ get_docker_containers_info() {
   ok
 }
 
+get_docker_sysconfig() {
+  try "Gathering Docker sysconfig"
+
+  if [ -e /etc/sysconfig/docker ]; then
+    cat /etc/sysconfig/docker > "$info_system"/docker/sysconfig
```

Comment:
There's a few things I'd like to see changed here:

- ensure the directory exists prior to writing (in case of reorder or other conditional steps)
- please use `cp`


```suggestion
    mkdir -p "${info_system}/docker"
    cp /etc/sysconfig/docker "$info_system"/docker/sysconfig-docker
```

Reply:
done